esp/services/ws_sql/antlr3c.cmake

```diff
@@ -41,7 +41,7 @@ endif()
 include(ExternalProject)
 ExternalProject_Add(
     antlr3c
-    DOWNLOAD_COMMAND tar xzf ${CMAKE_CURRENT_SOURCE_DIR}/website-antlr3/download/C/libantlr3c-3.4.tar.gz
+    DOWNLOAD_COMMAND tar xzf ${CMAKE_CURRENT_SOURCE_DIR}/libantlr3c/libantlr3c-3.4.tar.gz
```

Comment:
It is a little convoluted that the build untars a tarfile with the same files as what is already in the source.  But this is not really any different than it was.
The:
build/esp/services/ws_sql/antlr3c/src/libantlr3c-3.4/src
dir created from the tar xf cmd, and the:
HPCC-Platform/esp/services/ws_sql/libantlr3c/src
are identical.  Same for include dir and m4 dir.